lib/src/widgets/request_form_widget.dart

```diff
@@ -61,10 +62,17 @@ class RequestFormPage extends StatelessWidget {
               ? context.locale.add_request
               : context.locale.edit_request,
         ),
-        actions: [
-          if (onDelete != null)
-            IconButton(icon: const Icon(Icons.delete), onPressed: onDelete),
-        ],
+        actions: request != null
```

Comment:
too much nesting... Is it possible to create a getter and handle the empty case first?